<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistDlg</class>
 <widget class="QDialog" name="RegistDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>361</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="centerWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>430</width>
       <height>330</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>430</width>
       <height>330</height>
      </size>
     </property>
     <widget class="QPushButton" name="pushBtn_regist">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>200</y>
        <width>235</width>
        <height>35</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>235</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	
	color: rgb(255, 255, 255);
	border-style: solid;
	border-radius: 5px;	
	background-color: rgb(7,193,96);
}

QPushButton:hover{
	background-color: rgb(56,205,127);
}

QPushButton:pressed {
	background-color: rgb(7,193,96);
}</string>
      </property>
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_username">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>240</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	color: rgb(0, 0, 0);
	border-style: none;
	border-radius: 5px;	
	border-right:1px solid #3498db;
	border-top:1px solid #3498db;
	border-left:1px solid #3498db;
	border-bottom:1px solid #3498db;
	min-height: 20px;
}
QLineEdit:focus {
    border: 2px solid #3498db;
}</string>
      </property>
      <property name="text">
       <string>deroy</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>240</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	color: rgb(0, 0, 0);
	border-style: none;
	border-radius: 5px;	
	border-right:1px solid #3498db;
	border-top:1px solid #3498db;
	border-left:1px solid #3498db;
	border-bottom:1px solid #3498db;
	min-height: 20px;
}
QLineEdit:focus {
    border: 2px solid #3498db;
}</string>
      </property>
      <property name="text">
       <string>123456</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>用户名:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>53</x>
        <y>140</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>密  码:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(166,166,166);</string>
      </property>
      <property name="text">
       <string>注册账号</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>0</y>
        <width>76</width>
        <height>42</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushBtn_hide">
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border: none; /* no border for a flat push button */
	image: url(:/src/min.png); /* 最小化 */
	min-width: 34px;
	max-width: 34px;
	min-height: 40px;
	max-height: 40px;
}
QPushButton::hover {
	background-color: rgba(166,166,166, 50); 
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushBtn_close">
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border: none; /* no border for a flat push button */
	image: url(:/src/close.png); /* 最小化 */
	min-width: 34px;
	max-width: 34px;
	min-height: 40px;
	max-height: 40px;
}
QPushButton::hover {
	background-color: #f57575; 
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
